check_addon.py

```diff
@@ -349,3 +351,48 @@ def _logWarning(error_counter, warning_string):
     comments_warning.append(warning_string)
     error_counter["warnings"] = error_counter["warnings"] + 1
     return error_counter
+
+
+
+
+def get_addons(xml_url):
+    """
+        addon.xml for the target Kodi version
+    """
+    content = requests.get(xml_url).content
+    tree = ET.fromstring(content).getroot()
+
+    return {
+        a.get("id"): a.get("version")
+        for a in tree.findall("addon")
+    }
+
+
+def get_users_dependencies(xml_file):
```

Comment:
I think we can find a better name for this, as it has nothing to do with a `user`